core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -204,10 +196,6 @@ public int getContactPort() {
     return transactionManagerClass;
   }
 
-  public Isolation getIsolation() {
```

Comment:
Ditto. We need to change `DatabaseConfig.getIsolation()` to `ConsensusCommitConfig.getIsolation()`.